core/pom.xml

```diff
@@ -39,6 +39,15 @@
 Bundle-Name: Asset Share Commons - Core
 Bundle-Description: Asset Share Commons Core OSGi bundle containing AEM server-side logic
 Bundle-SymbolicName: com.adobe.aem.commons.assetshare.core
+# also detect scr descriptors from bundle fragments (https://docs.osgi.org/specification/osgi.cmpn/8.0.0/service.component.html#d0e30931)
+Service-Component: OSGI-INF/*.xml
+# reference packages from both fragment and this bundle: https://sling.apache.org/documentation/bundles/models.html#bundle-manifest-configuration
+Sling-Model-Packages: com.adobe.aem.commons.assetshare.components,\
```

Comment:
this can be simplified to just `com.adobe.aem.commons.assetshare` with the drawback of slightly decreased speed of https://github.com/apache/sling-org-apache-sling-models-impl/blob/6c3baab6bb71c329170fdeb0250db4c78a5cdb1d/src/main/java/org/apache/sling/models/impl/ModelPackageBundleListener.java#L114